src/lib/eslint/LWCEslintStrategy.ts

```diff
@@ -39,8 +42,8 @@ export class LWCEslintStrategy implements EslintStrategy {
 		return ENGINE.ESLINT_LWC;
 	}
 
-	isRuleKeySupported(key: string): boolean {
-		return key.startsWith(LWC_RULE_PREFIX);
+	isRuleKeySupported(): boolean {
+		return true;
```

Comment:
We haven't concluded if this method is still needed, returning `true` for now.

Reply:
Sorry, I forgot to follow-up on this. From what I remember about the flow, this is the method that causes a rule to be picked up or not when a category is chosen as a filter. I'll leave the actual fix to your discretion since you know the best after all the code changes - I think we may have to return true only when rule does not start with `@typescript`. If not, lwc engine may attempt to execute with a config that includes typescript rules.

Reply:
(something weird happening with Github - please ignore this comment if it shows up elsewhere)
From what I remember about the flow, this is the method that causes a rule to be picked up or not when a category is chosen as a filter. I'll leave the actual fix to your discretion since you know the best after all the code changes - I think we may have to return true only when rule does not start with `@typescript`. If not, lwc engine may attempt to execute with a config that includes typescript rules.